packages/qwik-city/buildtime/vite/dev-server.ts

```diff
@@ -62,6 +62,7 @@ export function configureDevServer(ctx: BuildContext, server: ViteDevServer) {
             requestCtx,
             params,
             routeModules,
+            {}, // TODO: Need to get actual platform object
```

Comment:
@adamdbradley @manucorporat The only part I'm confused by here is where/how the platform object would get pulled in and used. The assumption is that it would be set in the appropriate middleware. Is it pulled in and used by Qwik City automatically or do we need to manually get it and assign it?

Reply:
Since the dev server here is just our own express server then an empty object like this would be correct. This is a great PR, thanks!